### PR TITLE
CSI 5091 vg in storageclass

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: volume-group-operator/pkg/messages/controller_messages.go [hunk 1]
```diff
@@ -50,4 +50,5 @@ var (
 	AddVolumeToVolumeGroup                           = "Adding volume of persistentVolumeClaim to %s/%s volumeGroup"
 	AddedVolumeToVolumeGroup                         = "Successfully added volume of persistentVolumeClaim to %s/%s volumeGroup"
 	PersistentVolumeClaimIsNotInBoundPhase           = "PersistentVolumeClaim is not in bound phase, stopping the reconcile, when it will be in bound phase, reconcile will continue"
+	StorageClassHasVGParameter                       = "StorageClass contain parameter volume_group for claim %s/%s. volumegroup feature is not supported"
```
Comment (matancarmeli7) [inline volume-group-operator/pkg/messages/controller_messages.go:53]: can you add a `%s` after the `StorageClass`? that we will know which `storageClass` was meant in the log

File: volume-group-operator/controllers/persistentvolumeclaim/persistentvolumeclaim_controller.go [hunk 2]
```diff
@@ -88,6 +88,20 @@ func (r *PersistentVolumeClaimReconciler) isPVCNeedToBeHandled(reqLogger logr.Lo
 		reqLogger.Info(messages.PersistentVolumeClaimIsNotInBoundPhase)
 		return false, nil
 	}
+	IsSCHasVGParam, err := utils.IsPVCInStaticVG(reqLogger, r.Client, pvc)
```
Comment (matancarmeli7) [inline volume-group-operator/controllers/persistentvolumeclaim/persistentvolumeclaim_controller.go:91]:
```suggestion
	isSCHasVGParam, err := utils.IsPVCInStaticVG(reqLogger, r.Client, pvc)
```

File: volume-group-operator/pkg/messages/controller_messages.go (hunk 1 above)
Comment (ArbelNathan) [reply volume-group-operator/pkg/messages/controller_messages.go]: done

File: volume-group-operator/controllers/persistentvolumeclaim/persistentvolumeclaim_controller.go (hunk 2 above)
Comment (ArbelNathan) [reply volume-group-operator/controllers/persistentvolumeclaim/persistentvolumeclaim_controller.go]: done

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**